examples/transfer-simple.yaml

```diff
@@ -0,0 +1,68 @@
+# Simple transfer
+
+'@context':
+  '@vocab': https://w3id.org/valueflows#
+  xsd: http://www.w3.org/2001/XMLSchema#
+  time: http://www.w3.org/2006/time#
+  time:inXSDDateTimeStamp:
+    '@type': xsd:dateTimeStamp
+  alice: https://alice.example/
+  bob: https://bob.example/
+
+'@graph':
+
+  # initial resources
+
+  - '@id': alice:c7897c39-7f05-4a5d-a487-80e130a24345
+    '@type': EconomicResource
+    classifiedAs: http://www.wikidata.org/entity/Q89 # apples
+    currentQuantity:
+      qudt:unit: unit:Kilogram
+      qudt:numericValue: 670
+    trackingIdentifier: lot-4399jj93
+
+  # Alice gives Bob 200 kg apples.
+
+  - '@id': alice:e7482f32-25b5-4f79-934b-5d0db3a5b77d
+    '@type': Transfer
```

Comment:
can we just use `vf:Process` here since #389 will clarify that *give* and *receive* do NOT 'change existence' and based on meaning of those actions system should not consider this process as *transformation*?